lichess.py

```diff
@@ -41,10 +41,11 @@ def is_final(exception):
                           max_time=60,
                           interval=0.1,
                           giveup=is_final)
-    def api_get(self, path):
+    def api_get(self, path, raise_for_status=True):
```

Comment:
Unrelated to this PR, but would it be worth adding `timeout` as an additional, optional param. as well?